Buildings/Controls/OBC/FDE/DOAS/ExhaustFanController.mo

```diff
@@ -0,0 +1,153 @@
+within Buildings.Controls.OBC.FDE.DOAS;
+block ExhaustFanController
+  "This block manages start, stop, and speed of the exhaust fan."
+
+  parameter Real bldgSPset(
+   final unit="Pa",
+   final quantity="PressureDifference")=15
+    "Building static pressure set point";
+
+    // ---inputs---
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput supFanProof
+    "True when supply fan is proven on"
+      annotation (Placement(transformation(extent={{-142,34},{-102,74}}),
+        iconTransformation(extent={{-140,40},{-100,80}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput bldgSP
+    "Building static pressure"
+      annotation (Placement(transformation(extent={{-142,-28},{-102,12}}),
+        iconTransformation(extent={{-140,-84},{-100,-44}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput exhFanProof
+    "True when exhaust fan is proven on."
+      annotation (Placement(transformation(extent={{-142,-62},{-102,-22}}),
+        iconTransformation(extent={{-140,-20},{-100,20}})));
+
+     // ---outputs---
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput exhFanStart
+    "Command exhaust fan to start when true."
+       annotation (Placement(transformation(extent={{102,34},{142,74}}),
+         iconTransformation(extent={{100,40},{140,80}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput exhFanSpeed
+    "Exhaust fan speed command"
+      annotation (Placement(transformation(extent={{102,-16},{142,24}}),
+        iconTransformation(extent={{100,-80},{140,-40}})));
+
+  Buildings.Controls.OBC.CDL.Continuous.Sources.Constant bldgStaticSP(
+    k=bldgSPset)
+      "Building static pressure set point"
+        annotation (Placement(transformation(extent={{-40,2},{-20,22}})));
+  Buildings.Controls.OBC.CDL.Continuous.LimPID conPID(
+    k=0.0000001,
```

Comment:
There are very small number. I think you could normalize the measured pressure with the pressure setpoint, so that to the controller, the setpoint will be 1 and the measured value will be in the 0~1.

Reply:
We discussed some in our review the relevance of the CDL PID parameters with regards to HVAC applications. The values selected were only for purposes of eliciting an expected response in the model simulation. Since these parameter values will not translate to any vendors proprietary PID calculation they're only valid here.